recipe_scrapers/__init__.py

```diff
@@ -605,22 +606,44 @@ def scraper_exists_for(url_path: str) -> bool:
     return host_name in get_supported_urls()
 
 
-def scrape_me(url_path: str, **options: Any) -> AbstractScraper:
-    host_name = get_host_name(url_path)
+HEADERS = {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:86.0) Gecko/20100101 Firefox/86.0"}
+
 
-    try:
-        scraper = SCRAPERS[host_name]
-    except KeyError:
-        if not options.get("wild_mode", False):
-            raise WebsiteNotImplementedError(host_name)
+def check_scraper_allowed(url_path, host_name, **options: Any) -> bool:
+    """Checks the website's robots.txt file to see if scraping is allowed.
+    If scraping is allowed, returns true. Note that this does not check to see
+    when the last time robots.txt was checked, so this could lead to constant pings
+    on robots.txt"""
+    if not options.get("check_scraping", True):
+        return True
+    else:
+        url = urljoin(url_path, '/robots.txt')
+        resp = requests.get(url, headers=HEADERS)
+        page_data = resp.text
+        rp = urllib.robotparser.RobotFileParser()
+        rp.parse(page_data)
```

Comment:
Nitpick: lets factor-out the `resp` variable here:

```suggestion
        resp = requests.get(url, headers=HEADERS)
        rp = urllib.robotparser.RobotFileParser()
        rp.parse(resp.text)
```